lib/fluent/supervisor.rb

```diff
@@ -24,8 +24,10 @@
 require 'fluent/plugin'
 require 'fluent/rpc'
 require 'fluent/system_config'
+# require 'serverengine'
+require_relative '../../../serverengine/lib/serverengine'
 
-if Fluent.windows?
+if ServerEngine.windows?
```

Comment:
Don't use `ServerEngine.windows?` directly.
If it provides better way to know in windows or not, update `Fluent.windows?` method in fluent/env.rb to return result of `ServerEngine.windows?`.